resources/metadata/fortify.yaml

```
@@ -462,6 +462,31 @@ spec:
       - STAGES
       - STEPS
       default: maven
+    # Global maven settings, should be added to all maven steps
+    - name: projectSettingsFile
+      type: string
+      description: Path to the mvn settings file that should be used as project settings file.
+      scope:
+        - GENERAL
+        - STEPS
+        - STAGES
+        - PARAMETERS
+    - name: globalSettingsFile
+      type: string
+      description: Path to the mvn settings file that should be used as global settings file.
+      scope:
+        - GENERAL
+        - STEPS
+        - STAGES
+        - PARAMETERS
+    - name: m2Path
+      type: string
+      description: Path to the location of the local repository that should be used.
+      scope:
```

Please add an alias as here https://github.com/SAP/jenkins-library/blob/ce4be513625b184e98ab3660d9f313340b2981cf/resources/metadata/nexusUpload.yaml#L112 so that we can define the m2Path in general as maven -> m2Path
```suggestion
      aliases:
          - name: maven/m2Path
      scope:
```